README.md

```diff
@@ -11,4 +12,41 @@
 
 Opening the dashboard will give an overview of how drag queens performed in a particular season of RuPaul's Drag Race (default season 1). The center of the dashboard will contain a map marked with their hometowns. When hovered over, each marker will display basic information about the queen from that city, such as their name and age when they competed on the season. There will also be a table below the map that will show the queen's relative rankings. On the right side of the map, users will be able to see how well each queen performed in each episode of the season on a line graph, in terms of whether they placed relatively higher or lower compared to the other queens. Users can also filter for particular queens by name, age, or other categories like whether they won the season, if they were given the title of 'Miss Congeniality', or if they were the first to be eliminated on their season. When filtering for a particular queen, users can also see how many times a queen won, was safe, or was in the bottom on their season, both as counts and as percentages.
 
+## Usage
+To use the dashboard, simply visit the deployed link and explore the data. Users can interact with the map, table, and line graph to explore the data and filter by different categories.
+
+Deployed link: https://dragracerviz.herokuapp.com/
```

Comment:
This isn't the link to our app, it should be https://wispyisle.shinyapps.io/dragracer-dashboard/